src/main/java/htsjdk/variant/vcf/VCFFileReader.java

```diff
@@ -182,31 +211,13 @@ public IntervalList toIntervalList() {
     }
 
     public IntervalList toIntervalList(final boolean includeFiltered) {
-
-        //grab the dictionary from the VCF and use it in the IntervalList
-        final SAMSequenceDictionary dict = getFileHeader().getSequenceDictionary();
-        final SAMFileHeader samFileHeader = new SAMFileHeader();
-        samFileHeader.setSequenceDictionary(dict);
-        final IntervalList list = new IntervalList(samFileHeader);
-
-        int intervals = 0;
-        for (final VariantContext vc : this) {
-            if (includeFiltered || !vc.isFiltered()) {
-                String name = vc.getID();
-                final Integer intervalEnd = vc.getCommonInfo().getAttributeAsInt("END", vc.getEnd());
-                if (".".equals(name) || name == null)
-                    name = "interval-" + (++intervals);
-                list.add(new Interval(vc.getContig(), vc.getStart(), intervalEnd, false, name));
-            }
-        }
-
-        return list;
+        return fromVcf(this, includeFiltered);
```

Comment:
@yfarjoun actually... I'm a bit confused about the deprecations.  Were you intending to deprecate all the fromVCF methods since they're confusingly named?  There's still 2 with that name.

Reply:
sorry...you are correct. I added 2 more "toIntervalList" and deprecated the remaining "fromVcf"